v1/api/swagger/components/schemas/Host.yaml

```diff
@@ -199,4 +199,24 @@ allOf:
           type: string
         example:
           - 20:31:01:02:ad:f2:a8:e5
-          - 21:31:01:02:ad:f2:a8:e5
\ No newline at end of file
+          - 21:31:01:02:ad:f2:a8:e5
+      InstalledFWBaselineID:
+        type: string
+        format: uuid
+        description: >-
+          The ID of the firmware baseline that is installed on the host.
+      InstalledFWBaselineVersion:
+        type: string
+        description: >-
+          The version of the firmware baseline that is installed on the host.
+      AvailableFWBaselineID:
+        type: string
+        format: uuid
+        description: >-
+          The ID of the firmware baseline that is available for the host. This
+          field is only populated when there is a new firmware baseline available.
+      AvailableFWBaselineVersion:
+        type: string
+        description: >-
+          The version of the firmware baseline that is available for the host. This
+          field is only populated when there is a new firmware baseline available.
```

Comment:
These properties needs to be included under required

Reply:
These fields are computed and are for display only. They can't be updated.  Do they still need to be "Required"?

Reply:
They actually need to be required otherwise they will get generated with `omitempty`  and I don't want to hear this word anymore 😄 
cc: @mchuang3